guidelines/sc/22/pointer-target-spacing.html

```diff
@@ -0,0 +1,18 @@
+<section class="sc new">
+   
+    <h4>Pointer Target Spacing</h4>
+    
+    <p class="conformance-level">AA</p>
+	<p class="change">New</p>
+    
+    <p>For each target, there is an area with a width and height of at least 44 CSS pixels that includes it, and no other targets, except when:</p>
+    <ul>
+        <li><strong>Enlarge:</strong> A mechanism is available to change the CSS pixel size of all targets so that the width and height are each at least 44 CSS pixels;</li>
+        <li><strong>Inline:</strong> The target is in a sentence or block of text;</li>
+        <li><strong>Essential:</strong> A particular presentation of the target is essential to the information being conveyed.</li>
+    </ul>
+    <p class="note">This criteria has been formulated to increase the hit-area of small targets, but the group would like feedback from providers of touch-screen devices if there is another way of forming the criteria to better complement the tap-heuristics used.</p>
```

Comment:
remaining two instances of "criteria" > "criterion" ... sorry

Reply:
@patrickhlauke only found one? In "Note: This criteria..."

Hope they are all weeded out now - cannot find another instance of 'criteria' when searching https://raw.githack.com/w3c/wcag/wcag22-pointer-target-spacing/understanding/22/pointer-target-spacing.html